app/src/main/java/com/example/epamandroid/SimpleFragment.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.epamandroid
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.simple_fragment.simple_fragment_hello_button
+
+private const val BUTTON_TEXT: String = "buttonText"
```

Comment:
please move this into `companion object` inside SimpleFragment

Reply:
1. Also better to add in constant name words like `EXTRA` or `KEY`:  `BUTTON_TEXT_KEY`.
2. Constants should have value in `screaming snake case`

Reply:
done